### PR TITLE
Separating to submodules

### PR DESCRIPTION
**Pull request by j-tulforov**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,115 +1,21 @@
...
-    "skipLibCheck": true,                                /* Skip type checking all .d.ts files. */
-    "jsx": "react",
+    "target": "es5",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "declaration": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "noImplicitAny": false,
```
Comment (Tataklim) [inline tsconfig.json:10]: We can go hard and forbid using `any` :)

File: packages/banner/src/components/LidoStakeBanner.tsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React, { useEffect, useState } from 'react';
 import { Button, Box, Text, Link, useBreakpoint } from '@lidofinance/lido-ui';
 
-import { getStakeApy, STATIC_DEFAULT_APY } from '@/api/stakeApy';
+import { getStakeApy, STATIC_DEFAULT_APY } from '@common/stakeApy';
```
Comment (Tataklim) [inline packages/banner/src/components/LidoStakeBanner.tsx:4]: maybe it's better to do `common` folder as a separated package?
and for example put there some other heplers (just a suggestion for discussion)

File: tsconfig.json (hunk 1 above)
Comment (j-tulforov) [reply tsconfig.json]: done

File: README.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ $ yarn add @lidofinance/solido-sdk
 
 ## Using banner
 
-<img src="./src/assets/banner_horizontal.png" alt="Banner" />
+<img src="packages/banner/src/assets/banner_horizontal.png" alt="Banner" />
 
 ```ts
 import { LidoStakeBanner } from '@lidofinance/solido-sdk';
```
[the remark this answers is not in the document]
Comment (j-tulforov) [reply README.md]: yes, I said I'll change them than.

File: packages/banner/src/components/LidoStakeBanner.tsx (hunk 2 above)
Comment (j-tulforov) [reply packages/banner/src/components/LidoStakeBanner.tsx:4]: There is no too much files for making it separated package, It's not worthy

**Review by Tataklim — APPROVED** [review]

**Pull request merged**